conf.py

```diff
@@ -17,7 +17,7 @@
 
 
 # Data about this site
-BLOG_AUTHOR = {"en": "The MATE Team", "ca": "L'equip de MATE", "es": "El equipo de MATE", "nl": "Het MATE team"}
+BLOG_AUTHOR = {"en": "The MATE Team", "ca": "L'equip de MATE", "es": "El equipo de MATE", "ca": "L'équipe de MATE", "nl": "Het MATE team"}
```

Comment:
Use the correct language code for French, which is not "ca".

Reply:
a tipo :)